unittests/CppInterOp/FunctionReflectionTest.cpp

```diff
@@ -556,6 +558,7 @@ TEST(FunctionReflectionTest, IsVirtualMethod) {
   EXPECT_FALSE(Cpp::IsVirtualMethod(SubDecls[3])); // y()
 }
 
+#if !(__clang__ && __APPLE__)
 TEST(FunctionReflectionTest, JitCallAdvanced) {
```

Comment:
```suggestion
#ifdef __APPLE__
TEST(FunctionReflectionTest, DISABLED_JitCallAdvanced) {
#else
TEST(FunctionReflectionTest, JitCallAdvanced) {
#endif
```

We should use this pattern everywhere.

Reply:
Yes made changes according to this pattern everywhere